src/tests/ftest/launch.py

```diff
@@ -1083,6 +1083,7 @@ def _set_test_environment(self, servers, clients, list_tests, provider, insecure
                     os.environ["DAOS_TEST_SHARED_DIR"], "daos_test", "apps")
             os.environ["D_LOG_FILE"] = os.path.join(os.environ["DAOS_TEST_LOG_DIR"], "daos.log")
             os.environ["D_LOG_FILE_APPEND_PID"] = "1"
+            os.environ["D_LOG_FILE_APPEND_RANK"] = "1"
```

Comment:
Adding D_LOG_FILE_APPEND_RANK with existing D_LOG_FILE_APPEND_PID is going to double the number of log files, right? I should point out that it's already notoriously difficult to sort through the large number of logs from just D_LOG_FILE_APPEND_PID.

@phender Thoughts here?

Reply:
> Adding D_LOG_FILE_APPEND_RANK with existing D_LOG_FILE_APPEND_PID is going to double the number of log files, right? I should point out that it's already notoriously difficult to sort through the large number of logs from just D_LOG_FILE_APPEND_PID.
> 
> @phender Thoughts here?

no, it will not double the number of log files; it will just cause the existing log files to have both pid (due to D_LOG_FILE_APPEND_PID) and rank present in the resultant log filename.

Reply:
> I should point out that it's already notoriously difficult to sort through the large number of logs from just D_LOG_FILE_APPEND_PID.

this is kind of the purpose of this feature, to make it easier to sort through the large number of logs:) instead of opening each log and grepping for 'Setting rank to %d' to determine which engine rank what file corresponds to, instead with this option it will be in the filename itself... e.g.  daos_engine.log.54421.rank=12

Reply:
Oh, duh. The PID is unique to each rank :) Ignore my noise...
Yes, I think this will help